app/templates/partials/-output-data-help.hbs

```diff
@@ -3,7 +3,7 @@
   Licensed under the terms of the Apache License, Version 2.0.
   See the LICENSE file associated with the project for terms.
  --}}
-<h5 class="content-header">Raw Fields</h5>
+<h5 class="content-header">Raw Data</h5>
```

Comment:
Our PM wanted everything to be called Fields, I think. Let's keep this in if the section is called Raw Data and I'll reconcile them later.